README.md

```diff
@@ -84,6 +84,10 @@ In rare cases, you may want to also copy each dependency into the build product
 
 If you’ve modified your [Cartfile][], or you want to update to the newest versions of each framework (subject to the requirements you’ve specified), simply run the `carthage update` command again.
 
+### Transitive dependencies
+
+If the framework you want to add to your project has dependencies, Carthage will automatically fetch and build them for you. Once you've added a dependency and ran `carthage update`, Carthage will look for a Cartfile, fetch those repositories and build the associated frameworks that **you will have to drag yourself into your project**. You do not need the framework to have any spec file whatsoever.
```

Comment:
Also, I'd like to be opaque about the building part, because of the existence of the `--no-build` flag. We can just say that the dependencies-of-dependencies are “brought in” or “added” as well.